include/protocols/MeterSOS_S0.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * Plaintext protocol of S0 Pulse meter devices
+ *  --> https://www.sossolutions.nl/5-kanaals-s0-pulse-meter-op-usb
+ *
+ * The device is a USB s0 logger device.
```

Comment:
maybe just a little bit more description of the communication, at least "sending  a text-based protocol over a usb-connected uart interface"?
maybe the usb-IDs used by the device or a udev rule to identify it.